docs/DeveloperGuide.md

```diff
@@ -8,54 +8,84 @@
 
-* Manager#setPerson(String newName, Gender newGender, int newAge, int newHeight, int newOriginalWeight, int newCurrentWeight, int newTargetWeight, ActivityLevel newActivityLevel) - Calls the method below to set the attribute values of the `Person` object.
-* Person#setAll(String newName, Gender newGender, int newAge, int newHeight, int newOriginalWeight, int newCurrentWeight, int newTargetWeight, ActivityLevel newActivityLevel) - Updates the attribute values of the `Person` object.
-                                               
-Both operation are only executed by the `Manager` class of the `Logic` component. Only one instance of `Person` is ever instantiated. A default person is instantiated at the start with default attribute values and when the user enters their information for the first time during the set up, all the default values would be updated to the inputted values. Therefore, the command to enter the user information will result in a change in the attribute values and not the creation of a new `Person` object.
+**Main classes and methods used** 
 
-Given below is the example usage scenario and how the feature works.
+* [`Manager`](https://github.com/AY2021S1-CS2113-T14-4/tp/blob/master/src/main/java/seedu/dietbook/Manager.java): Stores a `Person` object.
+    * `Manager#setPerson(String newName, Gender newGender, int newAge, int newHeight, int newOriginalWeight, int newCurrentWeight, int newTargetWeight, FitnessLevel newFitnessLevel)`: Calls a method in `Person` class (listed below) to set the attribute values of the `Person` object.
+* [`Person`](https://github.com/AY2021S1-CS2113-T14-4/tp/blob/master/src/main/java/seedu/dietbook/person/Person.java): Stores all user information provided.
+    * `Person#setAll(String newName, Gender newGender, int newAge, int newHeight, int newOriginalWeight, int newCurrentWeight, int newTargetWeight, FitnessLevel newFitnessLevel)`: Updates the attribute values of the `Person` object.
+                                             
 
-Step 1. When the user launches the application for the first time. A default `Person` object will be initialised by `Manager` and the user will be prompted to enter their name.
+**Example usage scenario and how the feature work**<br/>
+_Summary_: Only one instance of `Person` is ever instantiated. A default person is instantiated at the start
+ with default attribute values and when the user enters their information for the first time during the set up, all the default values would be updated to the inputted values. Therefore, the command to enter the user information will result in a change in the attribute values and not the creation of a new `Person` object.
+
+**Step 1**. When the user launches the application for the first time. A default `Person` object will be initialised by `Manager` and the user will be prompted to enter their name.
  
+_Object Diagram:_<br/>
 ![Enter Info Step1](diagrams/Enter Info Step1.png)
```

Comment:
This previously added (not within this PR) URL looks broken because of spaces.

Reply:
These are actually images rather than links and they seem to be displayed as expected on the website but perhaps I’ll remove the spaces in the file name for standardisation. The URL links were indeed broken in the previous PR, I will fix them in this PR.

Reply:
Okay, looks like the urls work on the website but not in Github itself.

Reply:
I have renamed the files. Do let me know if there are any other errors.